src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Checkbox.js

```diff
@@ -28,6 +29,12 @@ export default class Checkbox extends React.Component<FieldTypeProps<boolean>> {
             return <Toggler checked={!!value} onChange={this.handleChange}>{label}</Toggler>;
         }
 
-        return <CheckboxComponent checked={!!value} onChange={this.handleChange}>{label}</CheckboxComponent>;
+        return (
+            <CheckboxComponent
+                active={!disabled}
```

Comment:
It's not very optimal that we called it everywhere else `disabled` but only on this component `active` :see_no_evil: Do you think it is a lot of work to change that? Should be quite easy because of flow, shouldn't it? (Same goes for the `Toggler` and `Switch` components).